platform-ui-skin/src/main/webapp/skin/less/social/skin/SocialPage/Style.less

```diff
@@ -110,12 +110,4 @@
       }
     }
   }
-
-  /* Make TopBar under drawers and other modals when opened */
```

Comment:
This comment indicates that the topbar will be under the drawer ? could drawers have wrong z-indexes ?

Reply:
The drawer z-index is 1035 like all other drawers and the topbar has 1030 z-index, so by default the topbar will be under this drawer.
By adding **z-index: @zindexStickyContainerUnder** the drawer z-index will be 990.